src/iterators.jl

```diff
@@ -0,0 +1,44 @@
+
+struct EachMember
+    m::Module
+    f::Union{Nothing, Function}
```

Comment:
I don't understand what `f` is for. In the `iterate` definitions below, I suspect that `f` is always `nothing`. You may be able to take out some code. Note that I didn't run it to confirm.

Reply:
I get it now. It's for `enumerate(OpenDSSDirect.EachMember(Lines, Lines.Name))`.

Reply:
It's a bit of an unusual iterator, so docs will help here.

Reply:
Added docs in 65cfc5f